spv/verifier.go

```diff
@@ -116,6 +116,7 @@ func VerifySPV() VerifyOpt {
 // you are using, some may return a HeaderJSON response others may return the blockhash.
 type PaymentVerifier interface {
 	VerifyPayment(context.Context, *Envelope, ...VerifyOpt) (*bt.Tx, error)
```

Comment:
we've still got an envelope here... plus do we need 2 functions in this interface? seems like only 1 is enough/needed

Reply:
True, want me to remove envelope code? I was going to leave it since it's likely being used by some.